socorro/external/es/crashstorage.py

```diff
@@ -319,6 +320,27 @@ def capture_crash_metrics(self, raw_crash, processed_crash):
             # later.
             self.config.logger.exception('something went wrong when capturing processed_crash_size')
 
+    def _index_crash_in_elasticsearch(self, connection, es_index, es_doctype, crash_document,
```

Comment:
Given that this crashstorage is only about Elasticsearch, I'd probably shorten this to `_index_crash`. You don't have to change it if you don't want to--either is fine.